utils/index.js

```diff
@@ -0,0 +1,2 @@
+export { get_shop_products as getShopProducts } from './get_shop_products';
+export { get_product_by_id as getProductById } from './get_product_by_id';
```

Comment:
Can we follow single name convention across files. In case you see any discrepancy please let me know.
Standard for project : variable name - camel case
Ideally file name should be snake case.